app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,40 @@
+class FoodsController < ActionController::Base
+  def index
```

Comment:
Recipes and Foods created by one user are not visible when any other user is logged in. Please make sure that all users can see foods and recipes created by other users but can not delete them.

Reply:
Updated foods and works as expected, but as for recipes they are still owner specific. We tried everything for recipes, please if you know what we are missing to let all users view recipes let me know.